misago/permissions/hooks/check_locked_category_permission.py

```diff
@@ -116,6 +116,4 @@ def __call__(
         return super().__call__(action, permissions, category)
 
 
-check_post_in_closed_category_permission_hook = (
-    CheckPostInClosedCategoryPermissionHook()
-)
+check_locked_category_permission_hook = CheckPostInClosedCategoryPermissionHook()
```

Comment:
Please change `CheckPostInClosedCategoryPermission` to `CheckLockedCategoryPermission` in this file.